cmd/dashboard/main.go

```diff
@@ -52,39 +70,45 @@ func main() {
 
 	pipelineClient, err := clientset.NewForConfig(cfg)
 	if err != nil {
-		logging.Log.Errorf("error building pipeline clientset: %s", err.Error())
-	} else {
-		logging.Log.Info("Got a pipeline client")
+		logging.Log.Errorf("Error building pipeline clientset: %s", err.Error())
 	}
 
 	k8sClient, err := k8sclientset.NewForConfig(cfg)
 	if err != nil {
-		logging.Log.Errorf("error building k8s clientset: %s", err.Error())
-	} else {
-		logging.Log.Info("Got a k8s client")
+		logging.Log.Errorf("Error building k8s clientset: %s", err.Error())
 	}
 
 	resource := endpoints.Resource{
 		PipelineClient: pipelineClient,
 		K8sClient:      k8sClient,
 	}
 
+	// Any functional changes to the the router mux should be reflected in endpoints/test-utils.test_go/func dummyServer()
 	logging.Log.Info("Registering REST endpoints")
 	resource.RegisterWeb(wsContainer)
 	resource.RegisterEndpoints(wsContainer)
 	resource.RegisterWebsocket(wsContainer)
-	resource.RegisterHealthProbes(wsContainer)
-	resource.RegisterReadinessProbes(wsContainer)
-
-	installedNamespace := os.Getenv("INSTALLED_NAMESPACE")
-	logging.Log.Infof("Searching for extensions in the namespace %s", installedNamespace)
-	resource.RegisterExtensions(wsContainer, installedNamespace)
+	resource.RegisterHealthProbe(wsContainer)
+	resource.RegisterReadinessProbe(wsContainer)
+	endpoints.RegisterExtensions(wsContainer)
 
+	logging.Log.Info("Creating controllers")
 	stopCh := signals.SetupSignalHandler()
-	resource.StartResourcesController(stopCh)
-	resource.StartNamespacesController(stopCh)
+	resyncDur := time.Second * 30
+	tektonInformerFactory := tektoninformers.NewSharedInformerFactory(resource.PipelineClient, resyncDur)
+	kubeInformerFactory := k8sinformers.NewSharedInformerFactory(resource.K8sClient, resyncDur)
+	// Add all tekton controllers
+	tektoncontroller.NewTaskRunController(tektonInformerFactory, stopCh)
+	tektoncontroller.NewPipelineRunController(tektonInformerFactory, stopCh)
+	// Add all kube controllers
+	kubecontroller.NewExtensionController(kubeInformerFactory, stopCh, wsContainer)
+	kubecontroller.NewNamespaceController(kubeInformerFactory, stopCh)
```

Comment:
The `stopCh` argument was leftover here from the refactor. We can remove it from the above functions because they don't use the argument (weird that Golang doesn't complain about this 😅 )